content/pt-br/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Acessando clusters
+weight: 20
+content_type: concept
+---
+
+<!-- overview -->
+
+Esse tópico fala sobre diversas maneiras de interagir com clusters..
+
+<!-- body -->
+
+## Acessando pela primeira vez com kubectl
+
+Se tiver acessando o Kubernetes API pela primeira vez, recomendamos usar o Kubernetes CLI, `kubectl`.
+
+Para acessar um cluster, precisa saber a localização do cluster e ter credenciais para acessá-lo. Geralmente, isso é configurado automaticamente quando você trabalha com um [Guia de introdução](/docs/setup/) [Guia de introdução] ou outra pessoa configurou o cluster e forneceu a você credenciais e uma localização. 
+
+Verifique o local e as credenciais que o kubectl conhece com esse comando:
+
+```shell
+kubectl config view
+```
+
+Muitos dos [exemplos](/docs/reference/kubectl/quick-reference/) fornecem uma introdução ao uso do `kubectl`, e a documentação completa pode ser encontrada na [kubectl referência](/docs/reference/kubectl/).
+
+## Acessando diretamente a API REST
```

Comment:
Required for the anchor in the end of the page to work properly
```suggestion
## Acessando diretamente a API REST {#directly-accessing-the-rest-api}
```

Reply:
Added anchor